src/components/MyClaimCreateView/MyClaimCreateView.tsx

```diff
@@ -102,9 +101,7 @@ class MyClaimCreateView extends Component<Props, State> {
               </div>
             </div>
             <SchemaEditor
-              schema={
-                sdk.CTypeUtils.getClaimInputModel(cType!) as common.Schema
-              }
+              schema={getClaimInputModel(cType!) as common.Schema}
```

Comment:
I have altered the getClaimInputModel to match the new CType class inside the demo client. Feels a little hacky, plus I had to remove some information. 

I will bring this up and hopefully see if there is a better way of doing it.